test/resources/az_volume.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/test/e2e/framework"
+	diskv1alpha2 "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/azuredisk/v1alpha2"
+	azDiskClientSet "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/clientset/versioned/typed/azuredisk/v1alpha2"
+	consts "sigs.k8s.io/azuredisk-csi-driver/pkg/azureconstants"
+)
+
+// should only be used for integration tests
+type TestAzVolume struct {
+	Azclient             azDiskClientSet.DiskV1alpha2Interface
+	Namespace            string
+	UnderlyingVolume     string
+	MaxMountReplicaCount int
+}
+
+// should only be used for integration tests
```

Comment:
@sunpa93 the comments here say that these should only be used for integration tests, but these are also used in e2e. Do you have any insights for what happened here?

Reply:
This comment was added back when we decided to separate the CRI based tests to a separate integration test suite but this plan was later retracted and this comment is therefore no longer relevant. It should be removed. 👍

Reply:
made the change in #393